package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "Module_4b",
+  "version": "1.0.0",
+  "description": "For this assignment, you're going to fork this repository to your own Github account and clone that repository locally. Inside the repo you'll find a placeholder for app.js, a placeholder trucks.js, and a 'public' directory. Inside the public directory you'll find an index.html file and a client.js file.",
+  "main": "app.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/pjrice4/Module_4b.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/pjrice4/Module_4b/issues"
+  },
+  "homepage": "https://github.com/pjrice4/Module_4b"
+}
```

Comment:
Package.json is missing a dependencies section, which should contain references to express, underscore, and body-parser. Without those, I'm unable to clone your repository and run it locally without having to debug errors and install the dependencies that are causing them by missing.